File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace WindowsApplication1
{
    public partial class Form1 : XtraForm
    {
        DataTable DT;
        
        public Form1()
        {
            InitializeComponent();

            DT = new DataTable();
            DT.Columns.Add("Text", typeof(string));
            DT.Rows.Add("Text");
            DT.Rows.Add("Text");

            gridControl1.DataSource = DT;

            myMemoEdit1.DataBindings.Add("EditValue",DT, "Text", true, DataSourceUpdateMode.OnPropertyChanged);

        }

        private void trackBarControl1_EditValueChanged(object sender, EventArgs e)
        {
            myMemoEdit1.Properties.ScrollWidth = trackBarControl1.Value;
            repositoryItemMyMemoEdit1.ScrollWidth = trackBarControl1.Value;
        }

        private void trackBarControl2_EditValueChanged(object sender, EventArgs e)
        {
            myMemoEdit1.Properties.ScrollButtonWidth = trackBarControl2.Value;
            repositoryItemMyMemoEdit1.ScrollButtonWidth = trackBarControl2.Value;
        }

        private void myMemoEdit1_EditValueChanged_1(object sender, EventArgs e)
        {
            BindingManagerBase Manager = BindingContext[DT];
            Manager.EndCurrentEdit();
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits XtraForm
		Private DT As DataTable

		Public Sub New()
			InitializeComponent()

			DT = New DataTable()
			DT.Columns.Add("Text", GetType(String))
			DT.Rows.Add("Text")
			DT.Rows.Add("Text")

			gridControl1.DataSource = DT

			myMemoEdit1.DataBindings.Add("EditValue",DT, "Text", True, DataSourceUpdateMode.OnPropertyChanged)

		End Sub

		Private Sub trackBarControl1_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles trackBarControl1.EditValueChanged
			myMemoEdit1.Properties.ScrollWidth = trackBarControl1.Value
			repositoryItemMyMemoEdit1.ScrollWidth = trackBarControl1.Value
		End Sub

		Private Sub trackBarControl2_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles trackBarControl2.EditValueChanged
			myMemoEdit1.Properties.ScrollButtonWidth = trackBarControl2.Value
			repositoryItemMyMemoEdit1.ScrollButtonWidth = trackBarControl2.Value
		End Sub

		Private Sub myMemoEdit1_EditValueChanged_1(ByVal sender As Object, ByVal e As EventArgs) Handles myMemoEdit1.EditValueChanged
			Dim Manager As BindingManagerBase = BindingContext(DT)
			Manager.EndCurrentEdit()
		End Sub
	End Class
End Namespace
